msteams-platform/toolkit/toolkit-v4/add-single-sign-on-vs.md

```diff
@@ -854,8 +854,8 @@ The following table lists the changes by Teams Toolkit:
 
    |**Type**|**File**|**Purpose**|
    |--------|--------|-----------|
-   |Create|`aad.template.json` under `template\appPackage`|Azure AD application manifest represents your Azure AD app. `template\appPackage` helps you to register an Azure AD app during local debug or provision.|
-   |Modify|`manifest.template.json` under `template\appPackage`|`webApplicationInfo` object is added into your Teams app manifest template. Teams requires this field to enable SSO. When local debugging or provisioning is triggered, you can see the change.|
+   |Create|`aad.template.json` under `template\appPackage`|Azure AD app manifest represents your Azure AD app. `template\appPackage` helps you to register an Azure AD app during local debug or provision.|
```

Comment:
In this case, let's retain the text "Azure AD application manifest" to disambiguate it from the next row which talks about (Teams) app manifest.

Reply:
Updated.